src/cloud/paths/measurements.yml

```diff
@@ -1,26 +1,42 @@
 summary: Bucket schemas
 get:
-  summary: List all measurement schemas of a bucket
+  summary: List measurement schemas of a bucket
+  description: |
+    Retrieves a list of explicitly defined measurement
+    [schemas]({{% INFLUXDB_DOCS_URL %}}/reference/glossary/#schema) for a bucket.
```

Comment:
```suggestion
    [schemas]({{% INFLUXDB_DOCS_URL %}}/reference/glossary/#schema) (`"schemaType": "explicit"`) for a bucket.
```

Reply:
The type value is `explicit`, so we should just say _explicit_ bucket schemas and link to a definition or explain "explicitly defined..."